src/wide/f32x8_t.rs

```diff
@@ -239,6 +239,11 @@ impl f32x8 {
             }
         }
     }
+
+    pub fn powf(self, exp: f32) -> Self {
+        let v: [f32; 8] = self.into();
+        v.map(|x| x.powf(exp)).into()
+    }
```

Comment:
Skia does a lot better than this:
https://github.com/google/skia/blob/2e47b3ccb65fe6ca0c84f4f6b6ffc33f243247c3/src/opts/SkRasterPipeline_opts.h#L1013-L1016

Though I guess this could be implemented in a separate PR.

Reply:
Good catch. We should do the same as Skia.